packages/adapter-postgres/src/TableTypes.ts

```diff
@@ -6,5 +6,5 @@ export default interface TableTypes {
   inherits?: string[];
   onCommit?: 'PRESERVE ROWS' | 'DELETE ROWS' | 'DROP';
   tablespace?: string;
-  primaryKey?: string | string[];
```

Comment:
:'(

Reply:
@kesne is there no way to make this keyof the table?

Reply:
There is, but I’m gonna punt for now. It causes a mismatch between the adapter table types and the migration / schema types I’ll probably tear the primary key out of the base table types and move to the migration and Schema directly